api/v1beta1/azuremanagedmachinepool_webhook_test.go

```diff
@@ -547,12 +547,12 @@ func TestAzureManagedMachinePoolUpdatingWebhook(t *testing.T) {
 			name: "Can't update SubnetName with error",
 			new: &AzureManagedMachinePool{
 				Spec: AzureManagedMachinePoolSpec{
-					SubnetName: pointer.StringPtr("my-subnet"),
+					SubnetName: pointer.String("my-subnet"),
```

Comment:
why all the changes from StringPtr to String?

Reply:
They're the same function, but the shorter name is now preferred. I think `pointer.StringPtr` was considered stuttering compared to `pointer.String`.

Reply:
Got it, thanks!